src/index.html

```diff
@@ -1,13 +1,415 @@
+                <div class="testimonial__photo-container">
+                  <img
+                    src="images/people/2.jpg"
+                    alt="Scarlett Hawkins"
+                    class="testimonial__photo"
+                  >
+                </div>
+
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Scarlett Hawkins</a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Senior Vice President
+                </div>
+
+                <p class="testimonial__about">
+                  My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+                </p>
+              </div>
+            </div>
+
+            <div class="testimonials__item grid__item grid__item--all-9-12">
+              <div class="testimonial">
+                <div class="testimonial__photo-container">
+                  <img
+                    src="images/people/3.jpg"
+                    alt="Max Henry"
+                    class="testimonial__photo"
+                  >
+                </div>
+
+                <h3 class="testimonial__name">
+                  <a href="#" class="link">Max Henry</a>
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--all-1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a href="tel:+1 234 5555-55-55" class="contact-us__info-link">
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a href="mailto:hello@miami.com" class="contact-us__info-link">
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--all-4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+
+                <a href="https://goo.gl/maps/JzmmBRd7p9XK8p2x5" class="contact-us__info-link">
+                  400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401
```

Comment:
Change formatting a bit, just a suggestion
```suggestion
                  400 first ave. n.
                  <br>
                  suite 700
                  <br>
                  Minneapolis, MN 55401
```